### DISCUSSION TITLE
Zincou ?

### DISCUSSION THREAD

**Discussion by smousseur** [lecousin/lc-spring-data-r2dbc#30] [General]
Ce serait un truc de fou si c'est vraiment le Zincou que je pense !
Je cherchais un moyen élégant et sans prise de tête pour faire du spring data relationnel en mode reactive et je tombe sur ton repos !
Ah au fait, c'est Malik pour info... Si c'est le Zincou que je pense... sinon et bien excusez moi du dérangement et bonne continuation.

**Comment by lecousin** [comment]
Salut Malik ! oui c'est moi 😆 mais c'est un repo qui date un peu, plus mis à jour depuis un moment...

**Comment by smousseur** [comment]
Hahahaaa excellent !
Comment ça va ? Ca roule ?

Sinon sur ce sujet précis, je vais faire autrement. Dans mon cas, c'est pas gênant si je fais 2 requêtes et si j'ai vraiment besoin d'un join, je ferais une requête native.
